packages/common/container-definitions/src/audience.ts

```diff
@@ -21,6 +20,23 @@ export interface IAudienceOwner extends IAudience {
 	 * @returns if a client was removed from the audience
 	 */
 	removeMember(clientId: string): boolean;
+
+	/**
+	 * Notifies Audience that current clientId has changed.
+	 * See IAudience.currentClientId and IAudienceEvents' "clientIdChanged" event for more details
+	 */
+	setCurrentClientId(clientId: string | undefined): void;
```

Comment:
Breaking... maybe we don't expect this to be implemented by consumers but even in that case I think it should be announced until we're type trimming and this is not visible.

Reply:
I'm not sure how type trimming helps - it will stay @alpha as it's used by every 1st party.
I've changed whole interface to internal. It's not used by 1st party.
Though I'm surprised it compiles - IProtocolHandler.audience is public member on alpha interface... Something does not add up here.

Reply:
IProtocolHandler is marked alpha though, so an alpha member is fine.